install/install.go

```diff
@@ -115,6 +115,34 @@ func checkPods(c *k8s.Client) int {
 	return 0
 }
 
+func checkTerminatingPods(c *k8s.Client) int {
+	cursor := [4]string{"|", "/", "—", "\\"}
+	fmt.Printf("😋   Checking if KubeArmor pods are stopped ...")
```

Comment:
Add a stop sign " 🛑 "

Reply:
have changed emogi to

Reply:
![Screenshot from 2023-01-24 15-53-59](https://user-images.githubusercontent.com/85174511/214268116-1a6f2ed4-262f-4b95-8a16-c0cd981ca1f5.png)

Reply:
@MdSahil-oss This is excellent! and also add this cross mark emoji "❌" for other services. 
For example:
```console
❌   Mutation Admission Registration ...
❌   KubeArmor Annotation Controller Service ...
❌   KubeArmor Annotation Controller Deployment ...
❌   KubeArmor Annotation Controller TLS certificates ...
❌   Service Account ...
❌   Cluster Role Bindings ...
❌   KubeArmor Relay Service ...
❌   KubeArmor Relay Deployment ...
❌   KubeArmor DaemonSet ...
❌   KubeArmor Policy Manager Service ...
❌   KubeArmor Policy Manager Deployment ...
❌   KubeArmor Host Policy Manager Service ...
❌   KubeArmor Host Policy Manager Deployment ...
🔴   Done Checking , ALL Services are stopped!                                                                                           
⌚️   Termination Time : 10.79347051s 
```
Since we are uninstalling the services so the cross mark emoji will be appropriate.

Reply:
@yasin-cs-ko-ak I have changes emogies to ❌.

Reply:
@MdSahil-oss Thank you for your contribution! This looks good.